includes/Core/Modules/Modules.php

```diff
@@ -256,7 +257,8 @@ function( Module $module ) {
 					}
 				);
 
-				$data['activeModules'] = array_keys( $non_internal_active_modules );
+				$data['activeModules']        = array_keys( $non_internal_active_modules );
+				$data['psiManuallyActivated'] = $this->manually_enabled( 'pagespeed-insights' ) && $this->options->get( Migration_1_46_0::OPTION_KEY_PSI_UPDATED );
```

Comment:
I think the logic here is slightly off, in combination with https://github.com/google/site-kit-wp/pull/4305/files#diff-b0980b9f904136a052c4068889aa1973f11d6a7fc5d1c784bf37329ac8c13f79R56. For example, right now it will evaluate to `false` when a site that is just starting with 1.46.0 (i.e. no migration) hasn't manually activated PSI, which for that version will always be the case (since you can't activate it anymore).

This will lead to the tooltip tour potentially showing, which is not correct.

I suggest we simplify this: We don't actually care for whether PSI was manually activated. We care for whether PSI is **not** manually activated and the current version was migrated to from a pre-1.46.0 version. I think this here could simply become:
```suggestion
				$data['showPSIForceActiveTour'] = ! $this->manually_enabled( 'pagespeed-insights' ) && $this->options->get( Migration_1_46_0::OPTION_KEY_PSI_UPDATED );
```

Then https://github.com/google/site-kit-wp/pull/4305/files#diff-b0980b9f904136a052c4068889aa1973f11d6a7fc5d1c784bf37329ac8c13f79R56 should check for whether that value is `true`.

Reply:
https://github.com/google/site-kit-wp/pull/4305/commits/6e07da780fa8d2be9e85a194dc55592b879d4b07